tests/cypress/e2e/features/cloudstorages.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2023-2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Cloudstorages', () => {
```

Comment:
Please, update existing file instead of making one more:  `tests/cypress/e2e/actions_tasks3/case_105_cloud_storage.js`